testing/fake-dom.js

```diff
@@ -71,6 +71,14 @@ export class FakeWindow {
     // Document.
     /** @const {!HTMLDocument} */
     this.document = self.document.implementation.createHTMLDocument('');
+    // NOTE: There was a change between chrome 119 and 120 where the above
+    // self.document.implementation.createHTMLDocument no longer inherits
+    // the baseURI of self.document. We explicitly set and force it thru
+    // a `<base>` element.
+    const base = this.document.createElement('base');
+    base.href = self.document.baseURI;
+    this.document.head.appendChild(base);
```

Comment:
I ran these tests with `--firefox` to make sure this addition doesn't break the tests on them, looks good! :+1:

Reply:
thanks for checking!